projects/fw_103_hw/src/main.c

```diff
@@ -0,0 +1,51 @@
+#include "adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+
+static void prv_button_interrupt_handler(const GpioAddress *address, void *context) {
+  uint16_t A6_data = 0;
```

Comment:
Variables shouldn't start with capitals, in fact normally variable names should be all lowercase.